features/old/proxy.feature

```diff
@@ -81,18 +73,17 @@ Feature: Proxy integration
     And I go to the integration page for service "one"
     Then I should see "A valid APIcast Policies endpoint must be provided"
 
-  @javascript
-  Scenario: Sorting mapping rules
+  @wip
```

Comment:
👀

Reply:
Yep.. it should pass, but it won't. I followed the same steps in the UI but the result is different and I haven't been able to figure out why.

Reply:
check https://github.com/3scale/porta/pull/2274/files#r537618015

> I followed the same steps in the UI but the result is different and I haven't been able to figure out why.

I followed the steps in the UI and I couldn't drag the mapping rules on the APIAP page. But it could be just me for whatever reason.

Reply:
Yeah, there is no drag feature anymore. They're re-ordered by `position` via editing.